week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -8,10 +8,20 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
-
+  constructor() {
+    this.myShoppingCart = []
+  }
   cartContains() {
     // Use console.log() to output everything contained in your cart
+    console.log(this.myShoppingCart)
+  }
+  addItem(item) {
+    this.myShoppingCart.push(item)
+  }
+  listItems() {
+    this.myShoppingCart.map((item)=> {
```

Comment:
You should use a `forEach` here because you do not return anything inside the `map` function